internal/command/output.go

```diff
@@ -63,9 +63,9 @@ func (c *OutputCommand) Outputs(statePath string) (map[string]*states.OutputValu
 	}
 
 	// Load the backend
-	b, backendDiags := c.Backend(nil)
+	b, backendDiags := c.backend(".", arguments.ViewHuman)
```

Comment:
This command does support both JSON and Human views so I think we should pass the chosen type as an argument instead of hard-coding to `ViewHuman` here?

Reply:
Good catch! In that case we may be potentially fixing a bug here; the original code [didn't pass a view type in](https://github.com/hashicorp/terraform/blob/e44b870b8bd6748a2c6773d7bd85195598e649c5/internal/command/output.go#L66).